addons/knobs/src/KnobStore.js

```diff
@@ -12,7 +15,12 @@ export default class KnobStore {
     this.store[key] = value;
     this.store[key].used = true;
     this.store[key].groupId = value.groupId;
-    this.callbacks.forEach(cb => cb());
+
+    // debounce the execution of the callbacks for 50 milliseconds
+    if (this.timer) {
```

Comment:
please use `lodash.debounce`. i believe it's already a dependency for knobs

Reply:
@danielduan not anymore: